src/controllers/auth.ts

```diff
@@ -90,7 +90,13 @@ export const loginUser = (req: Request, res: Response) => {
                         })
                     const accessToken = createAccessToken(email)
                     return res.status(200).json(
-                        { "jwtAccessToken": accessToken })
+                        {
+                            "username": user.username,
```

Comment:
It feels weird to me that we return all  this data of the user once we login. We also do that at the get profile endpoint is it really necessary?

Reply:
delete email response as it's redundant!